### PR TITLE
Bb 326 creating java virtuoso

### PR DESCRIPTION
**Pull request by jonathansmit7**

### CODE REVIEW

File: src/main/java/com/sysunite/weaver/connector/RelationProperty.java [hunk 1]
```diff
@@ -4,19 +4,30 @@
...
     this.originId = originId;
     this.predicate = predicate;
     this.targetId = targetId;
   }
 
+  public String getId() {
+    return id;
+  }
+
+  public void setId(String idd) {
```
Comment (malamili) [inline src/main/java/com/sysunite/weaver/connector/RelationProperty.java]: error, argument is idd

**Comment by malamili** [comment]
Im missing all the other changes

**Comment by malamili** [comment]
No more comments my part

**Comment by gvdent** [comment]
@bastbijl Can you provide the reasoning for the Relation and Value properties needing an ID? 

Modification of these interfaces also impact the Neo4j connector, so I want to make sure these modifications happen for the right reason and not because of a leaky abstraction.

**Comment by bastbijl** [comment]
@carvilsi asked the same thing. The id's are needed because virtuoso stores the content of the relationship inside a graph that carries the id as name.

**Pull request merged**